### PR TITLE
LPS-148060 [CP 2.0] - Add CP2ExportingActivationKeys tests

### PR DESCRIPTION
**Pull request by epark18**
https://issues.liferay.com/browse/LPS-148060

### CODE REVIEW

**Comment by epark18** [comment]
ci:test:sf

**Comment by barrosdennys** [comment]
LGTM!

**Comment by barrosdennys** [comment]
ci:forward

**Comment by barrosdennys** [comment]
ci:reopen

File: modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/osb-site-initializer-customer-portal-test/src/testFunctional/tests/CP2ExportingActivationKeys.testcase [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+			CP2NavigationMenu.goToProductActivation();
+
+			CP2NavigationMenu.clickOnProductButton(product = "Portal");
+		}
+
+		task ("Then the user should be able to assert the 'Export All Key Details (csv)' option in the Actions dropdown") {
+			CP2NavigationMenu.assertActionsOption(key_action = "Export All Key Details (csv)");
+		}
+	}
+
```
Comment (barrosdennys) [inline modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/osb-site-initializer-customer-portal-test/src/testFunctional/tests/CP2ExportingActivationKeys.testcase:59]: @epark18 isn't this scenario already covered by SuccessMessageWhenKeyIsDownloaded? I know that we have different test cases for it in the Test Map, but do you think that only 1 test case could cover both scenarios?

**Comment by epark18** [comment]
See https://github.com/pt-liferay-solutions/liferay-portal/pull/1970.

**Pull request closed without merging**